theblues/charmstore.py

```diff
@@ -1,7 +1,7 @@
 import logging
 
 import requests
-from requests.exceptions import HTTPError
+from requests.exceptions import HTTPError, RequestException, Timeout
```

Comment:
Yes, that's the standard for any mutliple import from a single package.

On Wed, Jun 10, 2015 at 11:03 AM Kyle Adams <notifications@github.com>
wrote:

> In theblues/charmstore.py
> <https://github.com/juju/theblues/pull/12#discussion_r32126942>:
>
> > @@ -1,7 +1,7 @@
> >  import logging
> >
> >  import requests
> > -from requests.exceptions import HTTPError
> > +from requests.exceptions import HTTPError, RequestException, Timeout
>
> Previously I didn't use that format unless the imports ran over the 80
> char width limit. Is that standard for all instances of multiple imports
> from the same module?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/juju/theblues/pull/12/files#r32126942>.
>